### PR TITLE
Import camera points

### LINKED ISSUE #486: Make sure all data has been migrated to the new site
Make sure they are all present on the new site, including all the camera points, photopoints, and photopoints images. 
Just to verify all data are presents it's better to go to the old site and compare with the new sites.

### PR DESCRIPTION
**Pull request by ramereth**
Resolves #486 

## Changes in this PR.
- [x] Update CSV data to match production data
- [x] Import Camera Points
- [x] Import Photo Points
- [x] Import Photo Point Images

## Testing this PR.

1. ``docker-compose build`` -- You need to update your docker instance to pull in wget!
1. ``docker-compose kill && docker-compose rm --all`` -- remove all old instances
1. ``docker-compose up -d postgres_host && sleep 10 && docker-compose up -d web``
1. ``docker-compose run web bash``
1. ``cd data_scripts``
1. ``./get_all.sh`` -- This might take a bit since it pulls in the images from the site.

### Expected Output.

Goto http://localhost:8000/sites/camp-creek/camera/ and you should see two Camera Points.

```
$ make test
[... test output ...]
```

@osuosl/devs

### CODE REVIEW

File: data_scripts/get_camera_points.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+if os.path.isdir("../sw_data/"):
+    datafile = '../sw_data/camera_points.csv'
+else:
+    datafile = '../csvs/camera_points.csv'
+
+
+# Title, Post date, Updated date, Marker, Coordinates, Lat, Long
+with open(datafile, 'r') as csvfile:
+    sitereader = csv.reader(csvfile)
+    for row in sitereader:
```
Comment (ramereth) [inline data_scripts/get_camera_points.py]: I probably should update the names here to reflect what we're importing. I copied the get_sites.py as a template.

**Comment by ramereth** [comment]
@osuosl/devs alright, this is ready for review!

File: data_scripts/get_photo_point_images.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+with open(datafile, 'r') as csvfile:
+    photoreader = csv.reader(csvfile)
+    for row in photoreader:
+        if row[0] != 'Nid':  # Skip the header
+
+            pp_image_id = row[0]
+            photo_point = PhotoPoint.objects.get(id=row[1])
+            date = datetime.strptime(row[2], "%a, %Y-%m-%d")
+            image_file = os.path.basename(row[3])
+            image = open("../media/pp_photos/" + image_file, 'r')
```
Comment (clementsm) [inline data_scripts/get_photo_point_images.py]: Looks like you are grabbing a file already on the server here? Perhaps a comment helping us know how the file got here?

Comment (ramereth) [reply data_scripts/get_photo_point_images.py]: That's being pulled in from the ``pull-pp-images.sh`` script. I can certainly add a comment here to clarify that.

**Pull request merged**